src/tests/test_append_blkalloc.cpp

```diff
@@ -66,16 +64,14 @@ class AppendBlkAllocatorTest : public testing::Test {
     BlkDataService& inst() { return homestore::data_service(); }
 
     virtual void SetUp() override {
-        m_token = test_common::HSTestHelper::start_homestore(
+        m_helper.start_homestore(
```

Comment:
All changes from now on tests is to change from static method to member method and thus has m_helper instantiated in all tests.